Quadriga/src/main/java/edu/asu/spring/quadriga/web/settings/WebsiteAboutEditController.java

```diff
@@ -35,15 +42,30 @@
 
     @Autowired
     private IRetrieveProjectManager projectManager;
-    
+
+    @Autowired
+    private AboutTextValidator validator;
+
+    /**
+     * Attach the custom validator to the Spring context
+     */
+    @InitBinder("aboutTextBean")
+    protected void initBinder(WebDataBinder binder) {
+        binder.setValidator(validator);
+    }
+
     @AccessPolicies({ @ElementAccessPolicy(type = CheckedElementType.PROJECT, paramIndex = 1, userRole = {
             RoleNames.ROLE_COLLABORATOR_OWNER, RoleNames.ROLE_PROJ_COLLABORATOR_ADMIN }) })
     @RequestMapping(value = "auth/workbench/projects/{ProjectId}/settings/editabout", method = RequestMethod.GET)
     public String editAbout(@PathVariable("ProjectId") String projectId, Model model, Principal principal)
             throws QuadrigaStorageException {
-        model.addAttribute("aboutText", aboutTextManager.getAboutTextByProjectId(projectId));
         IProject project = projectManager.getProjectDetails(projectId);
```

Comment:
I believe there is an annotation to inject the project right into the method, isn't there? Then you can remove this line.